src/gridstack.js

```diff
@@ -296,10 +296,12 @@
 
   var idSeq = 0;
 
-  var GridStackEngine = function(column, onchange, float, maxRow, items) {
+  var GridStackEngine = function(row, column, onchange, float, minRow, maxRow, items) {
```

Comment:
I do not want to have a number of rows, unlike columns which are fixed and special % CSS layout. Min/MaxRow is more than enough to get everything you need. Also maxRow is not that well supported so I don't want to expose 'row' which is a superset. Please take out.

Also adding new param to new GridStackEngine() means all the test cases that use need updating - please fix them (under spec/) and make sure `npm test` still works after your changes AND the demo/ apps run. I know it's a lot but something I will have to do otherwise and I don't have extra cycle for this feature right now.

Reply:
Im actually happy that it actually works. Will refactor tomorrow, I also need to get more familiar with the code. Also, is there any reasons why you use `var` instead of `let` and `const`?

Reply:
didn't write that legacy code! I'm actually converting to typescript (as I willl have hand pick all those changes since 1.0.0) as I hate JS spaghetti code.